ext/iperf3/iperf_api.c

```diff
@@ -6074,17 +6022,26 @@ int iperf_printf(struct iperf_test *test, const char *format, ...)
 			char linebuffer[1024];
 			int total_len;
 
+			linebuffer[0] = '\0';
+			if (ct) {
+				sprintf(linebuffer, "%s", ct);
+			}
+			if (test->title) {
+				sprintf(linebuffer + strlen(linebuffer),
+					"%s:  ", test->title);
+			}
+
 			va_start(argp, format);
-			r = vsnprintf(linebuffer, sizeof(linebuffer), format,
-				      argp);
+			r = vsnprintf(linebuffer + strlen(linebuffer),
+				      sizeof(linebuffer), format, argp);
```

Comment:
`sizeof(linebuffer)` looks wrong to me as previous line adds `strlen(linebuffer)` to `linebuffer`. Hence, I guess this should be `sizeof(linebuffer) - strlen(linebuffer)`.
Or did I miss something?

Reply:
You are right, a good catch. I will fix this, the big enough buffer / short titles in testing have safe us so far.